Index.tsx

```diff
@@ -67,6 +68,7 @@ const Index: React.FC = () => {
                 <img draggable={false} loading="lazy" className="demo-slide" width="300" height="200" alt="Slide 5" src="https://via.placeholder.com/300x200/333333"/>
                 <img draggable={false} loading="lazy" className="demo-slide" width="300" height="200" alt="Slide 6" src="https://via.placeholder.com/300x200/333333"/>
             </Slider>
+            <OnSlideExample/>
```

Comment:
Nice, we should probably do this for all examples in a separate PR. Things are getting messy real quick with the current setup.